impl/src/main/java/org/apache/myfaces/config/element/FacesConfig.java

```diff
@@ -93,9 +93,9 @@ public List<String> getProtectedViewsUrlPatternList()
      * @since 2.2.0
      * @return 
      */
-    public Map<String, ComponentTagDeclaration> getComponentTagDeclarations()
+    public List<ComponentTagDeclaration> getComponentTagDeclarations()
     {
-        return Collections.emptyMap();
+        return Collections.emptyList();
```

Comment:
If this is accepted, how should I update the description?  Keep it as `@since 2.2.0` or update to 4.0.0?  Maybe just mention the new return type?

Reply:
Good question.  I would say update to 4.0.0?